MediaBrowser.Channels.Adult.Beeg/Channel.cs

```diff
@@ -46,8 +47,7 @@ public string Description
 
         public async Task<ChannelItemResult> GetChannelItems(InternalChannelItemQuery query, CancellationToken cancellationToken)
         {
-            
-            _logger.Debug("cat ID : " + query.FolderId);
+            _logger.LogDebug("cat ID : {id}", query.FolderId);
```

Comment:
```suggestion
            _logger.LogDebug("cat ID : {Id}", query.FolderId);
```
PascalCase for these names.